packages/node/src/network/reqresp/handlers/pooledUserOpHashes.ts

```diff
@@ -10,33 +10,21 @@ import { ResponseError } from "../../../reqresp/response";
 import { RespStatus } from "../interface";
 
 export async function* onPooledUserOpHashes(
-  executors: Executors,
+  executor: Executor,
   relayersConfig: Config,
   req: ts.PooledUserOpHashesRequest
 ): AsyncIterable<EncodedPayload<ts.PooledUserOpHashes>> {
-  const { supportedNetworks } = relayersConfig;
-  let chainId: number | null = null;
-  for (const [_, id] of Object.entries(supportedNetworks)) {
-    const mempoolIds = networksConfig[id]?.MEMPOOL_IDS;
-    if (mempoolIds) {
-      for (const mempoolIdHex of mempoolIds) {
-        const mempoolId = deserializeMempoolId(mempoolIdHex);
-        if (mempoolId == deserializeMempoolId(req.mempool)) {
-          chainId = id;
-        }
-      }
-    }
-  }
-
-  if (chainId == null) {
+  const mempoolIds = networksConfig[relayersConfig.chainId]?.MEMPOOL_IDS;
```

Comment:
Should we consider a configuration variable to specify the canonical mempool id for chains that we do not support yet?

Reply:
added a configuration param. I'll need to add that config param in other places as well, but that will be part of this PR https://github.com/etherspot/skandha/pull/139